src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -156,7 +155,7 @@ export const ReserveConfiguration: React.FC<{ reserve: ComputedReserveData }> =
         <Box>
           <Alert sx={{ mt: '16px', mb: '40px' }} severity="error">
             <Trans>
-              {reserve.symbol} is frozen due to an Aave Protocol Governance decision.{' '}
+              {reserve.symbol} is frozen due to an Aave community decision.{' '}
```

Comment:
I noticed the `FrozenWarning` component has this same text that we might want to make consistent.

On a related note, is there a list of frozen assets somewhere? I was just curious what that warning looked like, but I couldn't find any assets that were frozen.

Reply:
Just created #996 to fix this across multiple places in the app